File: sdk/python/examples/quickstart.py
from legion_sdk import Legion
import os

base = os.getenv('LEGION_API_URL', 'http://localhost:3000')
key = os.getenv('LEGION_API_KEY', 'demo-key')

c = Legion(base, key)
print(c.health())
print(c.get_account('0x123'))
print(c.echo({'hello': 'world'}))

File: sdk/js/examples/quickstart.mjs
import { Legion } from '../dist/index.js';

const baseUrl = process.env.LEGION_API_URL || 'http://localhost:3000';
const apiKey = process.env.LEGION_API_KEY || 'demo-key';

const client = new Legion(baseUrl, apiKey);

console.log(await client.health());
console.log(await client.getAccount('0x123'));
console.log(await client.echo({ ping: 'pong' }));
